### PR TITLE
Feature/3.0/author

### PR DESCRIPTION
**Pull request by silvergrund**

### CODE REVIEW

File: library/Controller/Author.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+/**
+ * Class Author
+ * @package Municipio\Controller
+ */
+class Author extends \Municipio\Controller\Archive
+{
+    public function init()
+    {
+        $this->data['postType'] = get_post_type();
+        $this->data['template'] = !empty(get_field('archive_' . sanitize_title('author') . '_post_style', 'option')) ? get_field('archive_' . sanitize_title('author') . '_post_style', 'option') : 'collapsed';
```
Comment (Muckbuck) [inline library/Controller/Author.php:14]: Too long ternary

File: views/v3/author.blade.php [hunk 2]
```diff
@@ -1,57 +1,54 @@
...
-    {!!
-        paginate_links(array(
-            'type' => 'list'
-        ))
-    !!}
+    </div>
 
+    @includeIf('partials.sidebar.default', ['id' => 'content-area'])
 
+    @pagination([
```
Comment (Muckbuck) [inline views/v3/author.blade.php:39]: Might not work?

File: views/v3/partials/post/post-newsitem.blade.php [hunk 3]
```diff
@@ -1,35 +1,39 @@
...
+                @slot('top')
+
+                <span class="c-segment__top-date"> {{date_i18n('l d F Y', strtotime($post->postDate))}} </span>
+                @endslot
+                @if (municipio_get_thumbnail_source($post->id,array(400,250)))
+
+                        @image([
+                            'src'=> municipio_get_thumbnail_source($post->id, array(400,250)),
+                            'alt' => $post->postTitle,
+                            'classList' => ['']
```
Comment (Muckbuck) [inline views/v3/partials/post/post-newsitem.blade.php:26]: Remove classList

File: library/Controller/Author.php (hunk 1 above)
Comment (silvergrund) [reply library/Controller/Author.php]: fixed

File: views/v3/author.blade.php (hunk 2 above)
Comment (silvergrund) [reply views/v3/author.blade.php:39]: fixed

File: views/v3/partials/post/post-newsitem.blade.php (hunk 3 above)
Comment (silvergrund) [reply views/v3/partials/post/post-newsitem.blade.php]: fixed

**Comment by sebastianthulin** [comment]
@silvergrund Fixar du diffen?

**Pull request merged**